### DISCUSSION TITLE
关于书源规则

### DISCUSSION THREAD

**Discussion by chalinat** [xiaohucode/yidaRule#164] [General]
大佬很牛逼，竟然还有win版的，太喜欢了，但是提出一点小小的需求，书源规则能不能跟阅读那个共用啊，现在的书源太少了，自己又没有什么本事写书源。。。如果可以做到完全兼容阅读的书源那就太棒了

**Comment by ljun223** [comment]
看小说你直接用阅读不就好了，为什么用这个？

**Reply by chalinat** [reply]
1、我苹果手机，目前的源阅读软件版本低，新的书源兼容性不好，很多都没有了。而新版本收费昂贵，性价比不高
2、我的时间大多数在pc端，所以用这个，其次就是IOS版本了，但是根源还是在书源不多啊，如果书源多点妥妥的主流软件啊

**Reply by ljun223** [reply]
苹果会自签建议用香色书阁，益达就是香色停更后大佬不得已写的替代。还有iOS版源阅读，找到安装包自签也能用。
不想自签，在AppStore搜索千阅、读不舍手这俩软件也是兼容阅读书源的，直接就能下载用。

**Reply by chalinat** [reply]
有在用IOS的源阅读，我自签没问题，主要是源阅读的老大也不更新了，版本太老了，跟阅读的源仓库书源很对不兼容，很多涩涩书源打不开发现，也获取不到内容。他们整了个收费的内测版，我觉得不划算，没有加入。

**Reply by chalinat** [reply]
另外我觉得现在益达挺好的，功能不需要太多，够用就好，关键是要兼容好阅读3.0的源。

**Reply by ljun223** [reply]
要求兼容有点不太现实，毕竟阅读3.0和益达用的函数都不一样，兼容需要重写部分底层函数，工程太大。
毕竟，益达是参考了香色写出来的，可能还有部分宜搜·益达版基础，所以他的源其实更类似香色或者之前益达版宜搜。

**Reply by mabDc** [reply]
阅读很不错啊。我来支持阅读吧。看起来很大部分源码和这个分支相近。看起来是重写了。有能力的仍可以继续修改。https://github.com/mabDc/eso/tree/xiaohu

某些地方果然一样。侧重点不一样。

![image](https://github.com/xiaohucode/yidaRule/assets/19526331/b1b30376-fb52-4831-9290-ee4a593623bd)

**Reply by chenxingmoonset** [reply]
> 阅读很不错啊。我来支持阅读吧。看起来很大部分源码还是这个分支。有能力的仍可以继续修改。https://github.com/mabDc/eso/tree/xiaohu

佬，添加个ios支持吧😂

**Reply by mabDc** [reply]
> > 阅读很不错啊。我来支持阅读吧。看起来很大部分源码还是这个分支。有能力的仍可以继续修改。https://github.com/mabDc/eso/tree/xiaohu
> 
> 佬，添加个ios支持吧😂

我没苹果设备。当前代码几个平台都能编译呀。
还有这个仓库呢 https://github.com/xiaohucode/eso_source
